api/src/controllers/user.controller.ts

```diff
@@ -176,6 +177,7 @@ export class UserController {
       false,
       queryParams.noWelcomeEmail !== true,
       mapTo(User, req['user']),
+      req,
```

Comment:
I wonder if it would be better to change the arguments to this function. Since we are sending bits and pieces of req and also all of req, maybe it'd be best to shorten the argument list to just be `req` and then in the function we grab what we need out of req